cmd/start.go

```diff
@@ -272,6 +274,20 @@ var StartCmd = &cli.Command{
 			indexingServiceProofs = append(indexingServiceProofs, delegation.FromDelegation(dlg))
 		}
 
+		replicatorDir, err := mkdirp(dataDir, "replicator")
+		if err != nil {
+			return err
+		}
+
+		db, err := sqlitedb.New(filepath.Join(replicatorDir, "replicator.db"),
+			database.WithJournalMode("WAL"),
+			database.WithTimeout(5*time.Second),
+			database.WithSyncMode(database.SyncModeNORMAL),
+		)
+		if err != nil {
+			return fmt.Errorf("creating jobqueue database: %w", err)
+		}
+
```

Comment:
I really don't like that this setup code is leaking into our cmd package, but I don't see an alternative with the current design.